examples/lua_REPL/main.c

```diff
@@ -61,3 +61,4 @@ int main(void)
 
     return 0;
 }
+
```

Comment:
Murdock complains about this extra newline.

Reply:
I just fixed it. Next time feel free to push to my branch for little fixes like those.